data/spells/lib/spells.lua

```diff
@@ -296,3 +296,57 @@ function Creature:addDamageCondition(target, conditionType, listType, damage, ti
 	target:addCondition(condition)
 	return true
 end
+
+function Creature:addPartyCondition(combat, variant, condition, baseMana)
+	local position = self:getPosition()
```

Comment:
We use ``position`` in 
```lua
	local affectedMembers = {}
	for _, member in ipairs(members) do
		if member:getPosition():getDistance(position) <= 36 then
			affectedMembers[#affectedMembers + 1] = member
		end
	end
```
so I guess it would make more sense if I move this variable declaration to just above ``local affectedMembers = {}``